client.proto

```diff
@@ -10,19 +10,30 @@ enum msgType {
     INIT_MSG  = 1;
     GNB_MSG   = 2;
     AMF_MSG   = 3;
-    GNB_DISC  = 4;
-    GNB_CONN  = 5;
+    REDIRECT_MSG = 4;
```

Comment:
You should never add something new in between like this. Always append at the end.